unit1/save_run_past_model.py

```diff
@@ -0,0 +1,51 @@
+import gym
+from stable_baselines3 import PPO
+import os
+from stable_baselines3.common.evaluation import evaluate_policy
+from stable_baselines3.common.env_util import make_vec_env
+
+# if there is no any saved model, directly begin traning.
+# if there is a saved model, ask the user which model you want to use make further training.
+# train_model function parameters can be changed.
+# if you want to use another model change: models_dir, model, variables
+
+models_dir = "models/PPO"
+logdir = "logs"
+env = make_vec_env('LunarLander-v2', n_envs=16)
+
+model = PPO(
+        policy = 'MlpPolicy',
+        env = env,
+        n_steps = 1024,
+        batch_size = 64,
+        n_epochs = 4,
+        gamma = 0.999,
+        gae_lambda = 0.98,
+        ent_coef = 0.01,
+        tensorboard_log=logdir,
+        verbose=1)
+
+def train_model(timesteps=1000, iter=10, saved_model_fn_int=0):
+    #saved_model_fn_int: this parameter is for using past model.
+    TIMESTEPS = timesteps
+    for i in range(iter):
+        model.learn(total_timesteps=TIMESTEPS, reset_num_timesteps=False, tb_log_name="PPO")
+        saved_model_file_name = saved_model_fn_int + i * TIMESTEPS
+        model.save(f"{models_dir}/{saved_model_file_name}")
```

Comment:
you should use SB3 checkpoint callback probably

Reply:
My intent was the save all models that was trained. However, I looked the SB3 callbacks and I think there is no need to save all models because we are able to save the best model with using callbacks .This PR may be unnecessary. Thanks for your feedback.